run.sh

```diff
@@ -9,6 +9,7 @@
 # To test that try running `docker run --rm --env-file .env busybox /bin/sh -c 'echo $NODE_ECDSA_KEY_PASSWORD'`
 # This will output password with single quote. Not sure why this happens.
 optIn() {
+  ./srs_setup.sh
```

Comment:
do we need to update the version in L20 and L34?

Reply:
yea! that we need to update anyway in the release - I will add it now so we don't forget